src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Enduser/Services/ConnectionService.cs

```diff
@@ -0,0 +1,464 @@
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Altinn.AccessManagement.Core.Errors;
+using Altinn.AccessMgmt.Core.Models;
+using Altinn.AccessMgmt.Persistence.Repositories;
+using Altinn.AccessMgmt.Persistence.Repositories.Contracts;
+using Altinn.AccessMgmt.Persistence.Services;
+using Altinn.Authorization.ProblemDetails;
+
+namespace Altinn.AccessManagement.Enduser.Services;
```

Comment:
Service i API-laget altså. BFA (Backend-for-API) 🤔

Reply:
okey, så me i access mgmt APIet skal ikke forhode seg til me?